include/kernel/mempool_heap.h

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 #ifndef ZEPHYR_INCLUDE_MEMPOOL_HEAP_H_
+#define ZEPHYR_INCLUDE_MEMPOOL_HEAP_H_
```

Comment:
Shouldn't the guard be renamed to `ZEPHYR_INCLUDE_KERNEL_MEMPOOL_HEAP_H_` ?

Reply:
it could, but that is not what I am fixing here.. Trying to keep it minimal, I am sure there 100s of headers in the tree that do not follow this conventions